diff --git a/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index f5142af..fdfd432 100644
--- a/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -30,7 +30,7 @@ exports.networks = [
     { chainId: 63n, shortName: 'metc' },
     { chainId: 69n, shortName: 'okov' },
     { chainId: 71n, shortName: 'cfxtest' },
-    { chainId: 81n, shortName: 'joc' },
+    { chainId: 81n, shortName: 'sby' },
     { chainId: 82n, shortName: 'meter' },
     { chainId: 83n, shortName: 'meter-test' },
     { chainId: 88n, shortName: 'tomo' },
@@ -107,6 +107,8 @@ exports.networks = [
     { chainId: 3636n, shortName: 'BTNX' },
     { chainId: 3737n, shortName: 'csb' },
     { chainId: 3776n, shortName: 'astrzk' },
+    { chainId: 1261120n, shortName: 'azktn' },
+    { chainId: 6038361n, shortName: 'azkyt' },
     { chainId: 4002n, shortName: 'tftm' },
     { chainId: 4202n, shortName: 'lisksep' },
     { chainId: 4337n, shortName: 'beam' },
