diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index a3c670d..e004a15 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -112,6 +112,8 @@ exports.networks = [
     { chainId: 3636n, shortName: 'BTNX' },
     { chainId: 3737n, shortName: 'csb' },
     { chainId: 3776n, shortName: 'astrzk' },
+    { chainId: 1261120n, shortName: 'azktn' },
+    { chainId: 6038361n, shortName: 'azkyt' },
     { chainId: 4002n, shortName: 'tftm' },
     { chainId: 4157n, shortName: 'crossfi-testnet' },
     { chainId: 4202n, shortName: 'lisksep' },
@@ -144,7 +146,7 @@ exports.networks = [
     { chainId: 9728n, shortName: 'boba-testnet' },
     { chainId: 10000n, shortName: 'smartbch' },
     { chainId: 10001n, shortName: 'smartbchtest' },
-    { chainId: 10081n, shortName: 'joct' },
+    { chainId: 10081n, shortName: 'sby' },
     { chainId: 10200n, shortName: 'chi' },
     { chainId: 10242n, shortName: 'aa' },
     { chainId: 10243n, shortName: 'aat' },
